configs/sst_cs_net_perf_services-terminal-emulator.yaml

```diff
@@ -3,11 +3,8 @@ version: 1
 data:
   name: Terminal Emulator
   description: Terminal emulators and fullscreen terminal applications/libraries
-  maintainer: sst_cs_infra_services
+  maintainer: sst_cs_net_perf_services
   packages:
-    - dialog
-    - minicom
-    - xterm
```

Comment:
That leaves this config without any packages.  Either minicom/xterm need to be here (instead of sst_cs_net_perf_services-miscellany.yaml), or this file should be removed entirely.